saleor/product/tasks.py

```diff
@@ -4,8 +4,7 @@
 from ..discount.models import Sale
 from .models import Attribute, Product, ProductType, ProductVariant
 from .utils.attributes import generate_name_for_variant
-from .utils.variant_prices import (
-    update_product_minimal_variant_price,
+from .utils.variant_prices import (  # update_product_minimal_variant_price,
```

Comment:
This comment is necessary?

Reply:
This code should be refactored in next PR's I prefer left it commented instead of removing it. It is easier to look at how this code looks before my changes if the old code is commented instead of looking for it in repository/history.